app/controllers/application_controller.rb

```diff
@@ -7,9 +7,15 @@ class ApplicationController < ActionController::Base
   before_action :set_paper_trail_whodunnit
 
   def check_maintenance_status
-    if FeatureToggle.service_unavailable? && !%w[service-unavailable accessibility-statement privacy-notice cookies].include?(request.fullpath.split("?")[0].delete("/"))
-      redirect_to service_unavailable_path
-    elsif !FeatureToggle.service_unavailable? && request.fullpath.split("?")[0].delete("/") == "service-unavailable"
```

Comment:
I could have written the below logic without nesting single unless statements inside if/elsif statements, but I actually thought it was the clearest way to present the logic.

I wrote out a table of what behaviour we'd expect in each case and used that to determine the logic here.

![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/63662292/879b4fd8-a345-47a3-89dc-1a526f408113)